modules/button/css/stories.tsx

```diff
@@ -2,248 +2,254 @@ import React from 'react';
 import {storiesOf} from '@storybook/react';
 import withReadme from 'storybook-readme/with-readme';
 import {SystemIcon} from '@workday/canvas-kit-react-icon';
-import {editIcon, relatedActionsIcon} from '@workday/canvas-system-icons-web';
+import {activityStreamIcon, relatedActionsIcon} from '@workday/canvas-system-icons-web';
+import {css} from 'emotion';
 import README from './README.md';
 import './index.scss';
 
+const blueBackground = css({
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  backgroundColor: '#0875e1',
+  margin: '0 10px',
+  padding: '24px',
+  maxWidth: 'max-content',
+  borderRadius: '3px',
+  '.wdc-btn': {
+    margin: '0 12px',
+  },
+});
+
 storiesOf('CSS/Button', module)
   .addDecorator(withReadme(README))
-  .add('All', () => (
+  .add('Primary', () => (
     <div className="story">
-      <section>
-        <button className="wdc-btn wdc-btn-primary">Primary</button>
-        <button className="wdc-btn">Secondary</button>
-        <button className="wdc-btn wdc-btn-delete">Delete</button>
-      </section>
+      <h3>Large Primary</h3>
+      <button className="wdc-btn wdc-btn-primary">Primary Button</button>
+      <button className="wdc-btn wdc-btn-primary wdc-btn-disabled">Primary Button</button>
```

Comment:
Can you add a `disabled` attributed to all of the ones we're using the disabled class on? Right now they're all tab focusable.

Reply:
Actually all of our styles shouldn't require the class if the attribute is there, so you should be able to drop `wdc-btn-disabled` as well.